modules/rosapps/applications/screensavers/butterflies/lang/de-DE.rc

```diff
@@ -30,5 +30,5 @@ END
 
 STRINGTABLE DISCARDABLE
 BEGIN
-    IDS_DESCRIPTION         "Butterflies Bildschirmschoner"
+    IDS_DESCRIPTION         "Schmetterlingsbildschirmschoner"
```

Comment:
It's the name of the Screensaver so I think it can be kept as is. Any objections?

Reply:
Or either 'Schmetterlings-Bildschirmschoner' or 'Schmetterling Bildschirmschoner'. I'd prefer the 2nd one.